src/get_data.py

```diff
@@ -307,6 +307,18 @@ def create_user(
     session.add(user)
     site_group.users.append(user)
     session.commit()
-    print(user)
 
     return user
+
+
+# create a new site group
+def create_new_site_group(session: Session, site_group_name: str) -> SiteGroupSQL:
```

Comment:
could you use https://github.com/openclimatefix/pv-site-datamodel/blob/main/pvsite_datamodel/write/user_and_site.py#L29 instead. Then its not duplicated. So you just import this in sites_toolbox.py instead